src/components/quiz/QuizFeedback.tsx

```
@@ -0,0 +1,29 @@
+  return (
+    <div
+      className={`mt-6 rounded-lg border p-4 ${
+        isCorrect ? 'border-green-200 bg-green-50' : 'border-red-200 bg-red-50'
+      }`}
+    >
+      <p className="font-semibold">
+        {message || (isCorrect ? 'Correto! ✓' : 'Incorreto! ✗')}
+      </p>
+      <div
+        className="prose mt-2 max-w-none"
+        dangerouslySetInnerHTML={{ __html: explanationHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)